frontend/src/views/web/error-pages/error-404.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
@RishabhJain2018 : A big change you have to do here. Actually you have to make a module for 404 and call it using states. You don't need to write a whole html.
Briefly explain, every html is like a module which gets rendered in index.html, so every script and styles should only be declared in index.html. But you made a new html for 404 which is not a required solution. 
Please go through other html files like team or dashboard from which you can get an idea how to write a module.
Let me know if you need more help.